src/status_im/ui/screens/communities/views.cljs

```diff
@@ -0,0 +1,523 @@
+(ns status-im.ui.screens.communities.views (:require-macros [status-im.utils.views :as views])
+    (:require
+     [reagent.core :as reagent]
+     [re-frame.core :as re-frame]
+     [quo.core :as quo]
+     [status-im.i18n :as i18n]
+     [status-im.utils.core :as utils]
+     [status-im.utils.fx :as fx]
+     [status-im.utils.config :as config]
+     [status-im.constants :as constants]
+     [status-im.communities.core :as communities]
+     [status-im.ui.screens.home.views.inner-item :as inner-item]
+     [status-im.ui.screens.home.styles :as home.styles]
+     [status-im.ui.components.list.views :as list]
+     [status-im.ui.components.copyable-text :as copyable-text]
+     [status-im.react-native.resources :as resources]
+     [status-im.ui.components.topbar :as topbar]
+     [status-im.ui.components.icons.vector-icons :as icons]
+     [status-im.ui.components.colors :as colors]
+     [status-im.ui.components.chat-icon.screen :as chat-icon.screen]
+     [status-im.ui.components.toolbar :as toolbar]
+     [status-im.ui.components.bottom-sheet.core :as bottom-sheet]
+     [status-im.ui.components.react :as react]))
+
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide])
+  (re-frame/dispatch event))
+
+(defn community-list-item [{:keys [id description]}]
+  (let [identity (:identity description)]
+    [quo/list-item
+     {:icon                       (if (= id constants/status-community-id)
+                                    [react/image {:source (resources/get-image :status-logo)
+                                                  :style {:width 40
+                                                          :height 40}}]
+
+                                    [chat-icon.screen/chat-icon-view-chat-list
+                                     id
+                                     true
+                                     (:display-name identity)
+                                     ;; TODO: should be derived by id
```

Comment:
still TODO ?

Reply:
Yes, for the first implementation won't matter as we only support status community, but will implement in the next PR when we allow other communities to show.